src/components/page-donate/page-donate.tsx

```diff
@@ -19,54 +8,50 @@ export class PageDonate {
   @State() private amount?: number | null;
   @State() private showConfirmation: boolean = false;
   @State() private canNativeShare: boolean = false;
+  @State() private referral?: string | null;
 
   public componentWillLoad() {
     document.title = `Donate | Pizza to the Polls`;
-  }
+    const urlParams = new URLSearchParams(window.location.search);
```

Comment:
URLSearchParams has [no IE support](https://caniuse.com/?search=urlsearchparams). I don't know how much of the PttP audience that is, but probably not much. If it proves to be a problem, we can use [Polyfill.io](https://polyfill.io/) to polyfill the API.

Reply:
@ribordy & @jamesarosen if you don't want to use a polyfill, you could use this instead:
``` js
/**
 * Return the value of a URL parameter
 * @param url String 
 * @param key String
 */
const getUrlParam = (url, key) => {
  var results = new RegExp("[\?&]" + key + "=([^&#]*)").exec(url);
  if (results == null){
     return null;
  }
  return decodeURI(results[1]) || 0;
};

// Get the URL
const url = window.location.href;

// Usage
if (getUrlParam(url, "success") && getUrlParam(url, "amount_usd")) {
  // Do something
}
```

Reply:
Good catch! I didn't realize it didn't have total support. I had previously used the `query-string` npm package, but since this snippet is pretty straightforward, I'll just use it.

Reply:
Wasn't sure what you were using for url so obviously leave out that `const url = ... ` or whatever you don't need